xarray/tests/test_plot.py

```diff
@@ -1024,6 +1026,34 @@ def test_2d_coord_names(self):
         with raises_regex(ValueError, 'requires 1D coordinates'):
             self.plotmethod(x='x2d', y='y2d')
 
+    def test_plot_rgb_image(self):
+        DataArray(
+            easy_array((10, 15, 3), start=0),
+            dims=['y', 'x', 'band'],
+        ).plot.imshow()
```

Comment:
Is it possible to check that the colors have actually rendered correctly by introspecting deeper into the figures / axes that are generated by `imshow`?

Reply:
Well, it's almost certainly *possible* - but I have absolutely no idea where to start!

Reply:
I'm okay relying on matplotlib to plot colors properly. One sane option would be to mock `plt.imshow` to ensure it gets called properly, but we don't bother with that for our current tests. But that shouldn't get in the way of integration tests that verify that we can actually call matplotlib without any errors being raised.